FAQ.md

```diff
@@ -67,6 +67,14 @@ If you still want to enable this functionality, run `ujust toggle-ghns`
 
 Xwayland is disabled by default on GNOME, KDE Plasma, and Sway. Use `ujust toggle-xwayland` if you need it
 
+#### Why I can't install & use any Gnome extensions?
```

Comment:
should we call this "GNOME user extensions" everywhere? to avoid confusion

Reply:
We can do that